bundles/XliffBundle/src/ExportDataExtractorService/DataExtractor/DataObjectDataExtractor.php

```diff
@@ -72,7 +72,7 @@ public function extract(TranslationItem $translationItem, string $sourceLanguage
      *
      * @throws \Exception
      */
-    private function extractRawAttributeSet(TranslationItem $translationItem, string $sourceLanguage, array $targetLanguages, array $exportAttributes, bool $inherited): AttributeSet
+    private function extractRawAttributeSet(TranslationItem $translationItem, string $sourceLanguage, array $targetLanguages, array|null $exportAttributes, bool $inherited): AttributeSet
```

Comment:
We should constantly use the question mark.
```suggestion
    private function extractRawAttributeSet(TranslationItem $translationItem, string $sourceLanguage, array $targetLanguages, ?array $exportAttributes, bool $inherited): AttributeSet
```